### PR TITLE
PRO21 - create side menu component

### PR DESCRIPTION
**Pull request by Valdas-Stonkus**
Component created, waiting for your reviews.

### CODE REVIEW

File: src/Components/SideMenu/SideMenu.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
+$sideBarWidth: 250px;
```
Comment (SwallowEve) [inline src/Components/SideMenu/SideMenu.scss]: Valdas, would you like to replace  variables into variables.scss file, just to store all variables at one place?

**Review by SwallowEve — APPROVED** [review]
Nice job!!!

**Comment by Valdas-Stonkus** [comment]
> Valdas, would you like to replace  variables into variables.scss file, just to store all variables at one place? 

Sure, done.

File: src/Components/SideMenu/SideMenu.scss (hunk 1 above)
Comment (Valdas-Stonkus) [reply src/Components/SideMenu/SideMenu.scss]: Done.

**Review by prodevshub — APPROVED** [review]

**Pull request merged**